src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -143,6 +143,15 @@ impl<'a, 'tcx> TerminatorCodegenHelper<'a, 'tcx> {
             }
 
             if let Some((ret_dest, target)) = destination {
+                // Generate debug label
```

Comment:
Can we expand this comment, making it clear that this is to do with Yorick, and not regular debugging stuff?